src/plots/plots.js

```diff
@@ -1865,7 +1811,7 @@ plots.extendTrace = function(destTrace, srcTrace) {
  * The result is the original object reference with the new contents merged in.
  */
 plots.extendLayout = function(destLayout, srcLayout) {
-    return plots.extendObjectWithContainers(destLayout, srcLayout, plots.layoutArrayContainers);
+    return plots.extendObjectWithContainers(destLayout, srcLayout, Registry.layoutArrayContainers);
```

Comment:
`plots` vs. `Registry` was aliased but not updated in quite a few places. It should be otherwise equivalent.